cmake/OpenCVDetectWebNN.cmake

```diff
@@ -0,0 +1,22 @@
+ocv_clear_vars(HAVE_WEBNN)
+ocv_clear_vars(WEBNN_EMSDK)
+if(WITH_WEBNN)
+  set(WEBNN_HEADER_DIRS "$ENV{WEBNN_NATIVE_DIR}/out/Release/gen/src/include")
+  set(WEBNN_INCLUDE_DIRS "$ENV{WEBNN_NATIVE_DIR}/src/include")
+  set(WEBNN_LIBRARIES "$ENV{WEBNN_NATIVE_DIR}/out/Release/libwebnn_native.so")
+endif()
+
+try_compile(VALID_WEBNN
+      "${OpenCV_BINARY_DIR}"
+      "${OpenCV_SOURCE_DIR}/cmake/checks/webnn.cpp"
+      CMAKE_FLAGS "-DINCLUDE_DIRECTORIES:STRING=${WEBNN_INCLUDE_DIRS}\;${WEBNN_HEADER_DIRS}"
+                  "-DLINK_LIBRARIES:STRING=${WEBNN_LIBRARIES}"
```

Comment:
Set `-DLINK_DIRECTORIES` and `-DLINK_LIBRARIES` separately.

Reply:
Sorry, I did not understand this well. Do you mean that I should set `-DINCLUDE_DIRECTORIES` and `-DLINK_LIBRARIES` separately?